src/Compilers/Core/Portable/Binding/AbstractLookupSymbolsInfo.cs

```diff
@@ -181,12 +183,14 @@ private void AddArity(int arity)
                 hashSet.Add(arity);
             }
 
-            public void GetUniqueSymbolOrArities(out IArityEnumerable arities, out TSymbol uniqueSymbol)
+            public void GetUniqueSymbolOrArities(out IArityEnumerable? arities, out TSymbol? uniqueSymbol)
             {
                 if (this.HasUniqueSymbol)
                 {
                     arities = null;
+#nullable disable // Can '_uniqueSymbolOrArities' be null?
```

Comment:
Do we have an issue tracking these? Even just a link to an overall tracking item. Seems like a good idea to make clear that these fixes are temporary and not necessarily expected to exist forever going forward.

Reply:
All `#nullable disable` are temporary. Adding links for them all would produce a very large amount of noise.

Reply:
Linked